src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7080,12 +7080,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>The operator '{0}' requires a matching non-checked version of the operator to also be defined</value>
   </data>
   <data name="ERR_CannotBeConvertedToUTF8" xml:space="preserve">
-    <value>The input string cannot be converted into the equivalent UTF8 byte representation. {0}</value>
+    <value>The input string cannot be converted into the equivalent UTF-8 byte representation. {0}</value>
   </data>
   <data name="IDS_FeatureUTF8StringLiterals" xml:space="preserve">
-    <value>Utf8 String Literals</value>
+    <value>Utf-8 String Literals</value>
```

Comment:
Should it be capitalized?

Reply:
@AlekseyTs this question is for you. Let me know if you want me to change it.

Reply:
>>Should it be capitalized?

> this question is for you. Let me know if you want me to change it.

If the "official" name is all caps, I have no objection to the capitalization.

Reply:
👍 I will update this, and all of the compiler tests, to "UTF-8" for consistency

Reply:
Consider using lowercase for "string literals" to match the other feature names.